tests/e2e/utils/common.go

```diff
@@ -144,10 +144,10 @@ func cloudcoreDeploymentSpec(imgURL, configmap string, replicas int) *apps.Deplo
 								v1.ResourceName(v1.ResourceCPU):    resource.MustParse("100m"),
 								v1.ResourceName(v1.ResourceMemory): resource.MustParse("512Mi"),
 							},
-							Limits: v1.ResourceList{
+							/*Limits: v1.ResourceList{
```

Comment:
This is a good change. EdgeController is a cloud component and may not resource constraint environment

Reply:
If we don't need these test scenarios, why not delete them?

Reply:
OK. will update later.